WPMedia/fusion-news-theme-blocks
# THEMES-1906: Use correct info when calling signing service

## Description

#### Jira Ticket: [THEMES-1906](https://arcpublishing.atlassian.net/browse/THEMES-1906)

Every image originating in Photo Center needs to be signed and passed to the resizer using only the image ID. This PR makes sure that happens where users can manually enter URLs for images.

Note: checks on this PR will fail until the companion PR is merged in.

### PR Dependencies

- https://github.com/WPMedia/arc-themes-components/pull/268

## Test Steps

1. Checkout this branch `git checkout THEMES-1906`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/extra-large-manual-promo-block,@wpmedia/extra-large-promo-block,@wpmedia/hero-block,@wpmedia/large-manual-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/medium-promo-block,@wpmedia/small-manual-promo-block,@wpmedia/small-promo-block`
3. For each promo block (and the hero block) manually paste in a cloud front image URL (such as `https://cloudfront-us-east-1.images.arcpublishing.com/sandbox.themesinternal/L3E2MP3YGJAU5B3U7GXW54GSWM` or `https://cloudfront-us-east-1.images.arcpublishing.com/sandbox.themesinternal/TPP422VTKVGP7KOZUJ2Y6QXQOQ.jpg`).
4. Ensure the image correctly loads.


[THEMES-1906]: https://arcpublishing.atlassian.net/browse/THEMES-1906?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
